tt_metal/hw/firmware/src/ncrisc.cc

```diff
@@ -33,7 +33,7 @@ uint32_t noc_nonposted_writes_acked[NUM_NOCS] __attribute__((used));
 uint32_t noc_nonposted_atomics_acked[NUM_NOCS] __attribute__((used));
 uint32_t noc_posted_writes_num_issued[NUM_NOCS] __attribute__((used));
 
-CBInterface cb_interface[NUM_CIRCULAR_BUFFERS] __attribute__((used));
+CBInterface local_remote_cb_interface[NUM_CIRCULAR_BUFFERS] __attribute__((used));
```

Comment:
is this left over from the initial version w/ 3 pointers?  since there is no local/remote distinction, I think cb_interface is a better name

Reply:
Yes, will rename back to cb_interface